docs/implementation/lift-inference.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: docs
+title: Lift Inference
+---
+
+# Lift Inference
+
+Programs in the Effekt surface language are translated to the underlying Core language.
```

Comment:
Maybe we can mention that `Core` corresponds to `System Xi` in Brachthäuser 2020 and to `System C` in the 2022 OOPSLA paper?